connectors/sources/github.py

```diff
@@ -801,11 +799,20 @@ async def graphql(self, query, variables=None):
             msg = "Your Github token is either expired or revoked. Please check again."
             raise UnauthorizedException(msg) from exception
         except BadGraphQLRequest as exception:
-            if self.get_rate_limit_encountered(exception.status_code, exception):
-                await self._put_to_sleep(resource_type="graphql")
```

Comment:
Why do we no longer want to check rate limiting after a `BadGraphQLRequest` exception?

Reply:
This function raises `QueryError` instead of `BadGraphQLRequest` when rate limit is encountered.